### PR TITLE
add optionnal bo connect for config toolbar

### PR DESCRIPTION
**Pull request by Amalec78**
Ajout d'une option qui permet de désactiver la connexion au BO (CTRL + ALT +B).

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
:+1:

**Comment by fkroockmann** [comment]
:+1:

File: Config/config.yml [hunk 1]
```diff
@@ -17,3 +17,4 @@ bundle:
     # If you set debug true be sure to have correctly installed the javascript development envirronment see README.md => https://github.com/backbee/BbCoreJs
     ##
     debug: false
+    disable_bo: false
```
Comment (eric-chau) [inline Config/config.yml]: Can you call it ``disable_toolbar`` please?

File: Renderer/Helper/bbtoolbar.php [hunk 2]
```diff
@@ -46,7 +47,12 @@ public function __invoke()
                          ->getSection('settings');
         
         $wrapper = (isset($settings['wrapper_toolbar_id'])) ? $settings['wrapper_toolbar_id'] : '';
+
+        /* add option bo connect */
+        $config = $this->getRenderer()->getApplication()->getContainer()->get('bundle.toolbar.config')->getBundleConfig();
+        $disable_bo = (isset($config['disable_bo'])) ? $config['disable_bo'] : false;
```
Comment (eric-chau) [inline Renderer/Helper/bbtoolbar.php]: Use camel case for php variable please (``$disableToolbar``).

**Comment by eric-chau** [comment]
:+1:, please change name of some variables and I will merge it. Thank you @Amalec78

**Comment by Amalec78** [comment]
Voila

**Comment by mickaelandrieu** [comment]
@Amalec78 can you squash your commits ?

**Comment by Amalec78** [comment]
Oui !

**Comment by mickaelandrieu** [comment]
thank you @Amalec78

**Pull request merged**